README.md

```diff
@@ -393,6 +393,22 @@ Debug.prettyPrint(label: "array", array)
 Note:
 This can't be used to the operator-based API such as `p >>>`. (This is a Swift language's limitation)
 
+### Debug Settings
```

Comment:
I recommend to change section title to `LLDB Integration` and move to before `Recommend Settings`.

What do you think?

Reply:
It's better.
fixed it!